salt/lax/config/etc-nginx-sitesenabled-bot-lax.conf

```diff
@@ -0,0 +1,66 @@
+# the upstream component nginx needs to connect to
+upstream flask {
+    server unix:///tmp/bot-lax-uwsgi.sock;
+}
+
+{% if salt['elife.cfg']('cfn.outputs.DomainName') %}
+# close connection on those that don't specify a 'host' header
+# http://nginx.org/en/docs/http/request_processing.html
+server {
+    listen 80;
+    listen 443;
+    server_name "";
+    return 444;
+}
+{% endif %}
+
+# configuration of the server
+server {
+    # we always listen on port 8001
+    # in non-dev environments port 8001 is only available to internal traffic
+    {% if salt['elife.cfg']('cfn.outputs.DomainName') %}
+    listen      8001 ssl; # see /etc/nginx/nginx.conf for global ssl settings
+    server_name {{ salt['elife.cfg']('project.project_hostname') }}
+                {{ salt['elife.cfg']('project.full_hostname') }}
+                {{ salt['elife.cfg']('project.int_project_hostname') }}
+                {{ salt['elife.cfg']('project.int_full_hostname') }};
+    {% else %}
+    listen      8001;
+    server_name localhost;
+    {% endif %}
+
+    charset     utf-8;
+
+    access_log /var/log/nginx/bot-lax.access.log combined_with_time;
```

Comment:
same here for the full naming `bot-lax-adaptor.access.log`, I would keep it consistent